.github/workflows/test.yml

```diff
@@ -36,7 +36,7 @@ jobs:
       - name: Verify
         shell: python
         env:
-          INFRACOST_VERSION_EXPECTED: ${{ matrix.version }}
+          INFRACOST_VERSION_EXPECTED: "0.9.12"
```

Comment:
Will this need to be updated with each release?

If so can we get the latest version using the API or CLI here and check against that. We might need to pass in a `GITHUB_TOKEN` for the rate limits for this though.

FYI getting release from the GH CLI which I think is installed by default in actions:
```
gh release list --repo https://github.com/infracost/infracost --limit 1 | awk '{print substr($1, 2)}'
```

Reply:
No gh, but curl and jq are there